authors/apps/comments/migrations/0001_initial.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.1.5 on 2019-02-07 14:12
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('profiles', '0001_initial'),
+        ('articles', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Comment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('body', models.TextField()),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True, null=True)),
+                ('article', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='articles.Article')),
+                ('author', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='profiles.Profile')),
+            ],
+            options={
+                'ordering': ['created_at'],
+            },
+        ),
+        migrations.CreateModel(
```

Comment:
Your PR seems to have updated one of the previous migration files and this could break the code in staging and production. If you could please move the added migrations for comments to a new migration file that would be amazing.

Reply:
@missvicki I did not touch any existing models. These are new models for comments and replies and should not affect any existing setup.

Reply:
@missvicki, please clarify on this. I seem to agree with @pbkabali on this one.